tox.ini

```
@@ -36,4 +36,4 @@ deps =
 extras =
     typing
 commands =
-    mypy docs src winbuild Tests {posargs}
+    mypy conftest.py selftest.py setup.py docs src winbuild Tests {posargs}
```

To catch any new top-level scripts?
```suggestion
    mypy *.py docs src winbuild Tests {posargs}
```

---

Sadly and strangely, [that](https://github.com/radarhere/Pillow/commit/e69bc496d77aeb9ad486501305f5b8b3075b9087) [doesn't work.](https://github.com/radarhere/Pillow/actions/runs/10751715254/job/29819259203#step:8:17)